game.gd

```diff
@@ -27,23 +26,15 @@ func _ready() -> void:
 	_check_window_resize()
 	_purse = CoinPurse.new()
 	_upgrades = UpgradeSystem.new(_purse)
-
-	_stronghold = StrongholdScene.instantiate()
-	add_child(_stronghold)
-
 	_waves = Waves.new(enemy_spawn)
 	add_child(_waves)
 
 	var player = Player.create()
 
 	add_child(player)
+	add_child(Stronghold.instantiate())
 
-	_hud = (HUD.create(
-		_stronghold.get_health().get_health(),
-		_stronghold.get_health().get_max_health(),
-		_purse.get_coins(),
-		_upgrades
-	))
+	_hud = (HUD.create(100, 100, _purse.get_coins(), _upgrades))
```

Comment:
Although it does look a bit strange, we'll probably want to keep passing in the function call rather than hard values to the HUD, or pass a direct reference